### ISSUE TITLE
Add North Star Framework to OPL

### ISSUE THREAD

**Issue by kattekrab** [openpracticelibrary/openpracticelibrary#1914]
See pages 225-229 of DevOps Culture and Practice Enablement book 
also:
https://amplitude.com/north-star
https://productfolio.com/northstar-framework/
https://miro.com/miroverse/just-mads-north-star-metric-workshop/

**Comment by tdbeattie** [comment]
Hey @kattekrab - this has been on my list to do for a while... thank you for raising an issue. I'd be very happy to put a first version in for consideration.

**Comment by kattekrab** [comment]
Oh that'd be awesome @tdbeattie - and very very welcome :)

**Comment by kattekrab** [comment]
nudge @tdbeattie ;-)

**Comment by tdbeattie** [comment]
On it now! (Finally!) :-)

**Comment by valyonchev** [comment]
hahaha, I was about to offer my help.
Do you want to double act on it?

Val

e-mail: ***@***.***
mobile: +32.497.592.235
https://www.linkedin.com/in/valyonchev/

**Comment by tdbeattie** [comment]
Haha @valyonchev - great minds.....

Sure! I've just pushed first PR with base content (which is very slow/hanging on E2E tests.... may need @kattekrab to work some magic on the moderation side!)

Let's co-author next iteration!

**Comment by valyonchev** [comment]
Perfect. I will look at it first thing in the new year and suggest some
edits.
I have a few others on my list and will get them all out at the same time
(Open Space, One pagers)

Val

e-mail: ***@***.***
mobile: +32.497.592.235
https://www.linkedin.com/in/valyonchev/

**Comment by valyonchev** [comment]
I have made some additional edits and pushed to review (via Netlify)
There appears to be an issue with the image, but maybe it is just a Netlify
bug

Val

e-mail: ***@***.***
mobile: +32.497.592.235
https://www.linkedin.com/in/valyonchev/

**Comment by tdbeattie** [comment]
Awesome thanks @valyonchev. Looks much improved!

@kattekrab (who's is in a different year to us right now!) may be familiar with the image deploy issue.

**Comment by valyonchev** [comment]
Looks like there is something wrong with the merge. It failed across the
board.
We should look at it in the new year.

Happy New Year, everyone!

Val

e-mail: ***@***.***
mobile: +32.497.592.235
https://www.linkedin.com/in/valyonchev/

**Comment by kattekrab** [comment]
Practice card image - URL method isn't working, so I downloaded the pic, and uploaded it directly, and voila! Now working.
Merged, autotested, deployed! 

https://openpracticelibrary.com/practice/north-star-framework/

What a great way to start 2023!!! Now we can all go hunting the North Star 🌟 

Thank you @tdbeattie and @valyonchev .

Minor issue with the 2nd checklist point:

> "It represents vision and strategy. Our company's product and business strategy are"

Is that missing the end of the sentence?

<img width="725" alt="image" src="https://user-images.githubusercontent.com/172755/210155994-2df36de9-a71d-4eaf-9416-d9cec78b3304.png">
